app/src/main/res/values/dimens.xml

```diff
@@ -586,6 +586,9 @@
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
 
+  <!--Topic Summary-->
```

Comment:
Re-write as 
`<!-- TopicSummaryView -->`
One space before and after text

Reply:
Done